components/avatar/stories/avatar.stories.js

```diff
@@ -15,7 +15,7 @@ export default {
 				type: { summary: "string" },
 				category: "Component",
 			},
-			options: ["50", "75", "100", "200", "300", "400", "500", "600", "700"],
+			options: ["50", "75", "100", "200", "300", "400", "500", "600", "700", "800", "900", "1000", "1100", "1200", "1300", "1400", "1500"],
```

Comment:
ooo! Would you please make a card to add these new sizes to the table we have on the docs page on `main`? I was going to ask you to add them, and then realized the docs page on this branch is totally different than what's on `main`! We should make sure we reference the new sizes on that page eventually, once the big merge happens.


<img width="388" alt="Screenshot 2024-11-06 at 11 59 32 AM" src="https://github.com/user-attachments/assets/6e673850-4619-440e-914f-2a6de1f1eeff">